.travis.yml

```diff
@@ -14,20 +14,12 @@ matrix:
     # next LTS and master is next most important
     - node_js: "6"
       env: DEPLOY_VERSION=testing
-    # still in LTS maintenance until fall 2016 (also still in wide use)
```

Comment:
We can't drop 0.12 or 0.10. Dropping support is a breaking change and 2.x can't take breaking changes.

Reply:
It looks like test failures are coming from a transitive dependency of `npm-registry-couchapp`, I'm prepared to give them a pass for the time being to get this out the door.

Reply:
I just opened PR to `npm-registry-couchapp` to fix this: https://github.com/npm/npm-registry-couchapp/pull/273